src/webhook-proxy.ts

```diff
@@ -1,10 +1,11 @@
 import Logger from 'bunyan'
+// @ts-ignore
+// tslint:disable-next-line:no-implicit-dependencies
+import EventSource from 'eventsource' // This is needed for now to resolve error TS4023, until smee-client gets it's own types or get's rewritten in TypeScript.
```

Comment:
There is a better way. TS4023 means we need an explicit type definition. The type in question is `EventSource`. If we import it and use it explicitly as the return type, the error goes away. 

**`src/webhook-proxy.ts`**

```ts
import Logger from 'bunyan'
import SmeeClient from 'smee-client'
import EventSource from 'eventsource';

export const createWebhookProxy = (opts: WebhookProxyOptions): EventSource | undefined => {
  try {
    const smee = new SmeeClient({
      logger: opts.logger,
      source: opts.url,
      target: `http://localhost:${opts.port}${opts.path}`
    })
    return smee.start()
  } catch (err) {
    opts.logger.warn('Run `npm install --save-dev smee-client` to proxy webhooks to localhost.')
    return
  }
}
```